tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts

```diff
@@ -100,6 +107,130 @@ describe('E2E Testing for taqueria taquito plugin', () => {
 		}
 	});
 
+	test('Verify that taqueria taquito plugin can transfer amount of tezos using transfer command from one account to another', async () => {
+		try {
+			// Setting up docker container name
+			dockerName = 'local';
+			environment = 'development';
+
+			// 1. Start local sandbox
+			await exec(`taq start sandbox`, { cwd: `./${taqueriaProjectPath}` });
+
+			// 2. Get Bob's and Alice's account addresses
+			const initialContractList = await exec(`taq list accounts ${dockerName}`, { cwd: `./${taqueriaProjectPath}` });
+
+			const addressArray = itemArrayInTable(addressRegex, initialContractList);
+
+			// 3. Call transfer to transfer
+			await exec(`taq transfer ${addressArray[1]} --tez 1000`, { cwd: `./${taqueriaProjectPath}` });
+
+			// 4. Verify transfer results
+			const resultContractList = await exec(`taq list accounts ${dockerName}`, { cwd: `./${taqueriaProjectPath}` });
+			const amountArray = itemArrayInTable(amountRegex, resultContractList);
+			expect(amountArray[1]).toEqual('4000 ꜩ');
+		} catch (error) {
+			throw new Error(`error: ${error}`);
+		}
+	});
+
+	test('Verify that taqueria taquito plugin can throw a proper message for transfer amount of tezos using transfer command from one account to another if account does not enough tezos', async () => {
+		try {
+			// Setting up docker container name
+			dockerName = 'local';
+			environment = 'development';
+
+			// 1. Start local sandbox
+			await exec(`taq start sandbox`, { cwd: `./${taqueriaProjectPath}` });
+
+			// 2. Get Bob's and Alice's account addresses
+			const initialContractList = await exec(`taq list accounts ${dockerName}`, { cwd: `./${taqueriaProjectPath}` });
+
+			const addressArray = itemArrayInTable(addressRegex, initialContractList);
+
+			// 3. Call transfer to transfer
+			const transferResult = await exec(`taq transfer ${addressArray[1]} --tez 5000`, {
+				cwd: `./${taqueriaProjectPath}`,
+			});
+
+			// 4. Verify that taqueria throw an error for transfer
+			expect(transferResult.stderr).toContain('Error during transfer operation');
+			expect(transferResult.stderr).toContain('TezosOperationError: (temporary) proto.alpha.contract.balance_too_low');
+			expect(transferResult.stderr).toContain('No operations performed');
+		} catch (error) {
+			throw new Error(`error: ${error}`);
+		}
+	});
+
+	// TODO: NEED TO COMPLETE THIS TEST
+	test.skip('Verify that taqueria taquito plugin can transfer amount of tezos using call command from one account to another', async () => {
```

Comment:
For skipped tests I would remove them from the PR and then open a ticket with the tests in them that need to be completed

Reply:
I cleaned up all skipped tests for transfer/call command.

Reply:
Yes, call is an alias - https://taqueria.io/docs/plugins/plugin-taquito/